src/Microsoft.DotNet.Helix/Sdk/tools/xharness-runner/xharness-event-processor.py

```diff
@@ -125,9 +125,8 @@ def analyze_operation(command: str, platform: str, device: str, is_device: bool,
 
     global retry, reboot, android_connectivity_verified
 
-    # Kill the simulator when we fail to launch the app
     # Apps crashing can be infra failures apart form Apple devices where retries can be costly
```

Comment:
nit: typo and a small rewording
```suggestion
    # Apps crashing can be infra failures, retry except on Apple devices where retries can be costly due to small queue size
```